modules/machineset-azure-enabling-accelerated-networking-existing.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+
+[id="machineset-azure-enabling-accelerated-networking-existing_{context}"]
+= Enabling Accelerated Networking on an existing Microsoft Azure cluster
+
+You can enable accelerated networking on Azure by adding `acceleratedNetworking` to your machine set YAML file.
+
+.Prerequisites
+
+* Have an existing Microsoft Azure cluster where the Machine API is operational.
+
+.Procedure
+
+. List the machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+The machine sets are listed in the form of `<cluster-id>-worker-<region>`.
++
+.Example output
+[source,terminal]
+----
+NAME                                DESIRED   CURRENT   READY   AVAILABLE   AGE
+jmywbfb-8zqpx-worker-centralus1     1         1         1       1           15m
+jmywbfb-8zqpx-worker-centralus2     1         1         1       1           15m
+jmywbfb-8zqpx-worker-centralus3     1         1         1       1           15m
+----
+
+. For each machine set:
+
+.. Edit the custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc edit machineset <machine-set-name>
+----
+
+.. Add the following to the `providerSpec` field:
++
+[source,yaml]
+----
+providerSpec:
+  value:
+    ...
+    acceleratedNetworking: true <1>
+    ...
+    vmSize: <azure-vm-size> <2>
+    ...
+----
++
+<1> This line enables Accelerated Networking.
+<2> Specify an Azure VM size that includes at least four vCPUs. For information about VM sizes, refer to link:https://docs.microsoft.com/en-us/azure/virtual-machines/sizes[Microsoft Azure documentation].
+
+. To enable the feature on currently running nodes, you must replace each existing machine. This can be done for each machine individually, or by scaling the replicas down to zero, and then scaling back up to your desired number of replicas.
```

Comment:
Minimalism: "To enable the feature on currently running nodes, replace each existing machine.